llama/convert.py

```diff
@@ -134,5 +135,5 @@ def tiny_llama(model_path):
     weights, params = globals()[args.model_name](model_path)
     np.savez(str(model_path / "weights.npz"), **weights)
     if params is not None:
```

Comment:
We can remove the check for `None` here also since we always want the `config.json` to be written now.